system-design/orchestration/workload/workload-orchestration-edge-onboarding.md

```diff
@@ -0,0 +1,70 @@
+# Device Management Client Onboarding
+In order for the workload orchestration solution to manage the edge device's workloads, the device's management client must first complete onboarding.
+
+> Action: The details in this page are still under discussion and have not been finalized.
+
+**The onboarding process includes:**
+
+1. The end user provides the the workload orchestration web service's root URL to the device's management client
+1. The device's management client downloads the workload orchestration solution vendor's public root CA certificate using the [Onboarding API](../../margo-api-reference/workload-api/onboarding-api/rootca-download.md)
+1. Context and trust is established between the device's management client and the workload orchestration web service
+1. The device's management client uses the [Onboarding API](../../margo-api-reference/workload-api/onboarding-api/device-onboarding.md) to onboard with the workload orchestration solution
+1. The device's management client receives the client Id, client secret and token endpoint URL used to generate a bearer token.
+1. The device's management client receives the URL for the Git repository containing its desired state and an associated access token for authentication
+1. The [device capabilities](./device-capability-reporting.md) information is sent from the device to the workload orchestration web service using the [Device API](../../margo-api-reference/workload-api/device-api/device-capabilities.md)
+
+![Margo Management Interface Operational Flow Diagram (svg)](../../figures/margo-interface-generic.drawio.svg)
```

Comment:
The figure suggests the use of FIDO. Didn't we discuss in a call that this should be kept out of the margo spec?

Reply:
Added an "Action" to ensure this is not seen as finalized in this PR.